hudi-common/src/main/java/org/apache/hudi/common/table/log/InstantRange.java

```diff
@@ -34,10 +34,12 @@ public abstract class InstantRange implements Serializable {
 
   protected final String startInstant;
   protected final String endInstant;
+  protected final String rangeType;
 
-  public InstantRange(String startInstant, String endInstant) {
+  public InstantRange(String startInstant, String endInstant, String rangeType) {
     this.startInstant = startInstant;
     this.endInstant = endInstant;
+    this.rangeType = rangeType;
```

Comment:
The `rangeType` is only used by `toString()`.  Should the class name be used instead of adding `rangeType` in the constructor?

Reply:
Adjust as u say.